.github/workflows/build.yml

```diff
@@ -13,23 +13,30 @@ on:
 jobs:
     tests:
         runs-on: ubuntu-latest
+        continue-on-error: ${{ matrix.experimental }}
```

Comment:
As we would like to add support for SF7 with this PR, I'm not sure if this flag is needed 🤔